fml/BUILD.gn

```diff
@@ -225,3 +225,26 @@ executable("fml_unittests") {
     "//third_party/dart/runtime:libdart_jit",
   ]
 }
+
+config("benchmark_config") {
+  include_dirs = [ "//third_party/benchmark:benchmark_config" ]
```

Comment:
Please the use the `//benchmarking:benchmarking` target and its utilities for common benchmark setup.

Reply:
The problem with using `//benchmarking:benchmarking` is that it declares a `main` method. But, it conflict with main in `run_all_tests.cc` which is a part of `testing:dart` which also declares a `main` method.

Reply:
Done